### PR TITLE
Add GitHub issue and PR templates

### PR DESCRIPTION
**Pull request by jsturtevant**
fixes: #389 

Modified the standard templates and borrowed from https://github.com/kubernetes-sigs/cluster-api/tree/master/.github

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
```
Comment (detiber) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Add a note to indicate the below information is when reporting a bug for the CAPI image builder compared to the other projects hosted in this repo (the tooling in /v1, images/konfigadm, and images/kube-deploy)?

**Review by detiber — COMMENTED** [review]
Minor nit, otherwise lgtm

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jsturtevant) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: updated to reflect the different projects

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Pull request merged**